Documentation/gettingstarted/minikube.rst

```diff
@@ -33,6 +33,10 @@ Install kubectl & minikube
 
      minikube start --network-plugin=cni --memory=4096
 
+.. note:: The minikube node may have a taint set for ``NoSchedule``. Please run ``kubectl describe node minikube | grep Taints``. If you find a ``NoSchedule`` taint, you can remove using the command ``kubectl taint nodes minikube  node.kubernetes.io/not-ready:NoSchedule-``
```

Comment:
When does this happen? What version?

Reply:
The NoSchedule should go away when Cilium starts running.

Reply:
Agreed, Cilium will be scheduled on this node regardless the taint set on any node.